src/tasks/AndroidAppBuilder/Templates/monodroid-coreclr.c

```diff
@@ -198,34 +260,38 @@ mono_droid_runtime_init (const char* executable)
 
     chdir (g_bundle_path);
 
-    // TODO: set TRUSTED_PLATFORM_ASSEMBLIES, APP_PATHS and NATIVE_DLL_SEARCH_DIRECTORIES
+    struct host_runtime_contract host_contract = {
+        sizeof(struct host_runtime_contract),
+        NULL,    // context
+        NULL,    // get_runtime_property
+        NULL,    // bundle_proble
+        NULL,    // pinvoke_override
+        &external_assembly_probe };
 
-    const char* appctx_keys[3];
+    const char* appctx_keys[PROPERTY_COUNT];
     appctx_keys[0] = "RUNTIME_IDENTIFIER";
     appctx_keys[1] = "APP_CONTEXT_BASE_DIRECTORY";
-    appctx_keys[2] = "TRUSTED_PLATFORM_ASSEMBLIES";
+    appctx_keys[2] = "HOST_RUNTIME_CONTRACT";
```

Comment:
This makes the Android app on coreclr always go through the external assembly probe, which is more like the real scenario. Is there usefulness in keeping the TPA property / runtime loading from disk and letting that be enabled via some define that can be specified when building the app?

Reply:
I don't think so. We only loaded from disk before because it was the easiest path.

Reply:
I agree with Steve, although it would be interesting to see the difference in performance.